### ISSUE TITLE
"iRIC Maintainance" not shown in Start menu on Windows 10

### ISSUE THREAD

**Issue by kskinoue0612** [i-RIC/prepost-gui-installer#3]
On Windows 10, "iRIC Maintainance" is not shown in Start Menu, even the shortcut exists in Start Menu folder.
This problem do not happen on Windows 7.
See the screenshots below:

Short cuts in Start Menu folder:

![shot1](https://cloud.githubusercontent.com/assets/4031569/26766801/62c73cd4-49d3-11e7-9d1c-df777ae46c24.png)

On Windows 10:

![shot2](https://cloud.githubusercontent.com/assets/4031569/26766807/6f3dc0f0-49d3-11e7-8532-f65232669cfe.png)

On Windows 7:

![shot3](https://cloud.githubusercontent.com/assets/4031569/26766815/79260802-49d3-11e7-9234-cad744c91d74.png)

**Comment by Leon-Zhang** [comment]
Hi Keisuke,

Could you show me the content of the "iRIC Maintainance" short cut? 
I found there is even no corresponding exe file under installation folder.

Thanks,
Leon

**Comment by kskinoue0612** [comment]
I send you the screenshot.
In this case, I've installed iRIC to "C:\Users\Keisuke\iRIC_USGS_dev".

![shot](https://user-images.githubusercontent.com/4031569/26914030-d5d6ec14-4c59-11e7-848f-7250e89b642b.png)

**Comment by Leon-Zhang** [comment]
I'm using iric version downloaded from here:
http://i-ric.org/en/downloads
And it didn't create the folder you shown in screenshot.

So seems I need use iRIC_USGS version?

**Comment by kskinoue0612** [comment]
Oh, I've got it. iRIC 3.0 installer is not available on English web site temporarily.

Please go to Japanese download page, and download "iRIC version3 beta 版 (64bit) 170607".
http://i-ric.org/ja/downloads

**Comment by Leon-Zhang** [comment]
I have turned to Japanese language page, and the download link appears disabled.
![ss_iric_3](https://user-images.githubusercontent.com/2606646/27062333-46ae97d8-5025-11e7-82b8-e0705585796a.png)


And seems I found another bug on iric website:

I can't login into my account on Japanese language page but only able to login from
English language page; and the reset password email been like following:

 様

iRICウェブサイトにて、パスワードの再通知申請を
受け付けました。
 様の現在のパスワードは、以下の通りとなって
います。
ご確認ください。

-------------------------------------------------

■ パスワード

-------------------------------------------------

今後とも、iRICをよろしくお願いいたします。


iRIC事務局 http://i-ric.org/ja/

Seems I'm only able to register and login on English version, and English version and Japanese version are using different database?

**Comment by kskinoue0612** [comment]
You are right, English version and japanese version are now using different database. To download files from Japanese site, please register again on Japanese website, from the following URL.

http://i-ric.org/ja/registration

I understand that you do not like this, but this is the specification of the website.

**Comment by Leon-Zhang** [comment]
Thanks for confirm!
